modules/datastore/src/DataDictionary/AlterTableQueryBase.php

```diff
@@ -14,43 +14,31 @@ abstract class AlterTableQueryBase implements AlterTableQueryInterface {
 
   /**
    * Keep track of whether this query has already been executed or not.
-   *
-   * @var bool
    */
   private bool $executed = FALSE;
 
   /**
    * Database connection.
-   *
-   * @var \Drupal\Core\Database\Connection
    */
   protected Connection $connection;
 
   /**
    * Strptime-to-MySQL date format converter.
-   *
-   * @var \PDLT\ConverterInterface
    */
   protected ConverterInterface $dateFormatConverter;
 
   /**
    * Query table.
-   *
-   * @var string
    */
   protected string $table;
 
   /**
    * Query fields.
-   *
-   * @var array
```

Comment:
It's fine that we remove `@var` here, but we also don't have inline documentation of how to define fields or indexes. We should add that in the `__construct()` docblock, but that can be a follow-up.

Reply:
OK will do

Reply:
See #4369